content/hi/docs/reference/glossary/container-env-variables.md

```diff
@@ -0,0 +1,16 @@
+---
+title: कंटेनर पर्यावरण चर (Container Environment Variables)
```

Comment:
**कंटेनर एनवायरनमेंट वेरिएबल्स** Use this and dont give English under bracket

Reply:
@CloudShiva7cr  To maintain the consistent look of the page  give english in bracket as well.

Reply:
कंटेनर एनवायरनमेंट वेरिएबल्स (Container Environment Variables)

Reply:
Sure. Add english title bracket (Container Environment Variables) 
Thanks!!